localstack/services/lambda_/invocation/runtime_executor.py

```diff
@@ -119,6 +121,13 @@ def __init__(self, message: str):
         super().__init__(message)
 
 
+@dataclasses.dataclass
+class LambdaPrebuildContext:
+    docker_file_content: str
+    context_path: Path
```

Comment:
(nit: path might be sufficient to reduce repetition but some extra verbosity does not hurt)

Reply:
Yes, maybe. The context_path is really of the container build context, and the prebuild context is more of a dto about the prebuild process itself. It's not perfectly named, but we can change that in the future, if we find a better way.